src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php

```diff
@@ -224,11 +236,25 @@ public function sendEmailAction(Request $request)
         try {
             /** @var UserInterface $user */
             $user = $this->findUser($request->get('user'));
+        } catch (EntityNotFoundException $ex) {
+            // do not report when there is no user, because this could be used for username enumeration attacks
+            return new JsonResponse(null, Response::HTTP_NO_CONTENT);
+        } catch (UserNotInSystemException $ex) {
+            if ($this->logger instanceof LoggerInterface) {
+                $this->logger->error($ex->getMessage(), ['exception' => $ex]);
```

Comment:
this is an expected exception this should not end in an error log.

Reply:
Yeah, we should probably handle different exceptions differently. Like if the mailer can't reach the mail server. This could be considered an error. But having this as logging is already better than nothing.